charts/newrelic-infrastructure-beta/templates/secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if not (include "newrelic.licenseCustomSecretName" .) }}
+{{- if (include "newrelic.licenseKey" . | required "licenseKey or customSecretName and customSecretLicenseKey must be set" ) }}
```

Comment:
Templates should only template. `required` should be in `helpers`. The helper should provide directly the value needed to template.

Reply:
As above, I still prefer the required outside (it will highlight better the line where the error is). Is there a recommendation on the contrary?

Reply:
Let's continue this conversation [above](https://github.com/newrelic/helm-charts/pull/624/files#r774742800)